suggest/preexisting_conditions.txt

```diff
@@ -4229,12 +4156,12 @@ growth hormone secreting pituitary adenoma (GHSPA)
 guanidinoacetate methyltransferase deficiency (GMD)
 gum cancer (GC)
 gummatous syphilis (GS)
-gynatresia
+gynatresia or vaginal stenosis
 gynecomastia
 gyrate atrophy (GA)
 haemonchiasis
 haemophilus meningitis (HM)
-hair disease (HD)
+hair disease (HD) or balding
```

Comment:
I'm surprised HD should stay for this one

Reply:
removed.